api/src/test/java/org/pmiops/workbench/db/dao/UserServiceTest.java

```diff
@@ -41,52 +49,73 @@
 @AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
 public class UserServiceTest {
 
-  private final String EMAIL_ADDRESS = "abc@fake-research-aou.org";
-
-  private Long incrementedUserId = 1L;
+  private static final String EMAIL_ADDRESS = "abc@fake-research-aou.org";
+  private static final int MOODLE_ID = 1001;
 
   // An arbitrary timestamp to use as the anchor time for access module test cases.
   private static final long TIMESTAMP_MSECS = 100;
-  private static final FakeClock CLOCK = new FakeClock();
+  private static final Instant START_INSTANT = Instant.ofEpochMilli(TIMESTAMP_MSECS);
+  private static final FakeClock PROVIDED_CLOCK = new FakeClock(START_INSTANT);
+  private static DbUser providedDbUser;
+  private static WorkbenchConfig providedWorkbenchConfig;
 
-  @Autowired private UserDao userDao;
-  @Mock private AdminActionHistoryDao adminActionHistoryDao;
-  @Autowired private UserDataUseAgreementDao userDataUseAgreementDao;
-  @Mock private FireCloudService fireCloudService;
-  @Mock private ComplianceService complianceService;
-  @Mock private DirectoryService directoryService;
+  @Autowired private FireCloudService mockFireCloudService;
+  @Autowired private ComplianceService mockComplianceService;
+  @Autowired private DirectoryService mockDirectoryService;
+  @Autowired private UserServiceAuditAdapter mockUserServiceAuditAdapter;
 
-  private UserService userService;
+  @Autowired private UserService userService;
+  @Autowired private UserDao userDao;
 
-  private DbUser testUser;
+  @TestConfiguration
+  @Import({UserService.class})
+  @MockBean({
+    AdminActionHistoryDao.class,
+    FireCloudService.class,
+    ComplianceService.class,
+    DirectoryService.class,
+    UserServiceAuditAdapter.class
+  })
+  static class Configuration {
+    @Bean
+    @Scope(ConfigurableBeanFactory.SCOPE_SINGLETON)
```

Comment:
singleton should be the default, the convention is generally to not include that

Reply:
It is. I wanted to remind myself it was deliberate since we've been struggling with these lately.

Reply:
I'd prefer a consistent codebase, I don't think this is going to be useful for the next reader who comes along - they will probably wonder why this line specifies singleton, and then 10 lines later another leaves the default scope (also singleton)

Reply:
👍